sapi/fpm/fpm/fpm_main.c

```diff
@@ -1584,7 +1580,7 @@ int main(int argc, char *argv[])
 
 #ifdef ZTS
 	tsrm_startup(1, 1, 0, NULL);
-	tsrm_ls = ts_resource(0);
+	(void)ts_resource(0);
```

Comment:
`php_ignore_value`?

Reply:
Same as above. There is no warning when the [`ts_resource()`](https://github.com/php/php-src/search?q=ts_resource&unscoped_q=ts_resource) result is no used.